src/components/_ui/cta/Cta.module.scss

```
@@ -1,2 +1,101 @@
+@import '@styles/constants';
+
+$fold-size: 50px;
+$default-animation-duration: .4s;
+
 .cta {
+    position: relative;
+    border: 0;
+    font-size: clamp(1.6rem, 1.6vw, 2rem);
+    font-family: inherit;
+    text-transform: uppercase;
+    box-sizing: border-box;
+    overflow: hidden;
+    cursor: pointer;
+
+    &::before {
+        content: "";
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: calc(100% - $fold-size);
+        height: $fold-size;
+    }
+
+    &::after {
+        content: "";
+        position: absolute;
+        top: 0;
+        right: 0;
+        border-style: solid;
+        border-left: none;
+        border-top: none;
+        border-right: solid $fold-size transparent;
+    }
+
+    &.yellow {
+        &::before {
+            background-color: $color-yellow;
+        }
+
+        &::after {
+            border-bottom: solid $fold-size $color-red;
+        }
+
+        background: linear-gradient(0deg,
+            $color-yellow 0%,
+            $color-yellow calc(100% - $fold-size),
```

```suggestion
            $color-yellow calc(100% - ($fold-size - 1px)),
```

na resolução do meu monitor ficou de boa, na resolução da tela do notebook ficou com um maldito 1 pixel transparente haha. Eu baixei aqui e testei essa gambiarra e deu boa. Ele corta um pedacinho do efeito de fold, mas é imperceptível, melhor do que ficar com uma linha preta